include/grpc/grpc.h

```diff
@@ -351,6 +351,19 @@ typedef struct grpc_op {
   } data;
 } grpc_op;
 
+/* Propagation bits: this can be bitwise or-ed to form propagation_mask for
```

Comment:
Yeah... I wanted to talk to @jboeuf before I went ahead and added it - and I didn't get a chance yet (fallout from late-on-a-Friday-afternoon-decision-making-in-my-cubicle).